### PR TITLE
Add failover site

### PR DESCRIPTION
**Pull request by steventlamb**
I pruned the source to produce a very limited page and used wget to grab a static copy of the site

### CODE REVIEW

File: src/failover_website/index.html [hunk 1]
```diff
@@ -1 +1,49 @@
-Hello, world.
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <!-- vendor.css contains styles which override bootstrap styling.  It must come after main.css -->
+    <link rel="stylesheet" href="static/css/main.css">
+    <link rel="stylesheet" href="static/css/vendor.css">
+    <link rel="stylesheet" type="text/css" href="http://cloud.typography.com/7429932/717646/css/fonts.css" />
```
Comment (hectcastro) [inline src/failover_website/index.html]: Omit this font from the failover site?

File: src/failover_website/index.html [hunk 2]
```diff
@@ -1 +1,49 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <footer class="visible-xs">
+        <div class="container">
+            <img class="section-about-logo pull-left" src="static/img/logo-nycparks.png" srcset="/static/img/logo-nycparks@2x.png 2x">
```
Comment (hectcastro) [inline src/failover_website/index.html]: Retina graphic is missing for me.

Comment (steventlamb) [inline src/failover_website/index.html]: Ah wget must not be able to handle this feature. I'll make sure the 2x graphics make it in too.

**Comment by hectcastro** [comment]
@azavea-bot rebuild

**Comment by steventlamb** [comment]
@hectcastro comments addressed, still works

File: src/failover_website/index.html (hunk 2 above)
Comment (maurizi) [inline src/failover_website/index.html]: You may have to remove the leading `/` from the `srcset` option too

**Comment by designmatty** [comment]
+1 Looks good.

**Pull request merged**